packages/kad-dht/test/rpc/handlers/get-value.spec.ts

```diff
@@ -79,7 +79,7 @@ describe('rpc - handlers - GetValue', () => {
     const value = uint8ArrayFromString('world')
     const record = new Libp2pRecord(key, value, new Date())
 
-    await datastore.put(utils.bufferToRecordKey(key), record.serialize().subarray())
+    await datastore.put(utils.bufferToRecordKey('//record', key), record.serialize().subarray())
```

Comment:
Is it expected that a prefix would look like this? Can we use something here that better reflects what this might be in the wild? Some people might look through tests to see how things work, and we should set them up for success.

Reply:
I've filled out the log prefix